<?xml version="1.0" ?>
<launch>

  <arg name="data_dir"		default="$(find WRS_Dataset)"/>
  <arg name="intrinsic"		default="realsense_intrinsic.json"/>
  <arg name="camera_frame"	default="map"/>

  <node name="feeder"
	pkg="aist_depth_filter" type="image_feeder.py" output="screen">
    <param name="data_dir"	value="$(arg data_dir)"/>
    <param name="intrinsic"	value="$(arg intrinsic)"/>
    <param name="camera_frame"	value="$(arg camera_frame)"/>
  </node>

  <include file="$(dirname)/run.launch"
  	   ns="feeder">
    <arg name="camera_info_topic"	value="camera_info"/>
    <arg name="image_topic"		value="image"/>
    <arg name="depth_topic"		value="depth"/>
  </include>

  <node name="$(anon rviz)" pkg="rviz" type="rviz" output="screen"
	args="-d $(dirname)/feeder.rviz"/>

</launch>
